src/hooks/account/useGetAccountFromApi.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+import { getAccountFromApi } from 'apiCalls';
+import { AccountType } from 'types';
+
+export const useGetAccountFromApi = (address: string | null) => {
```

Comment:
use swr for this. It makes it easyer with null

Reply:
Added. 👍